src/main/java/com/rovio/ingest/WriterContext.java

```diff
@@ -252,6 +258,7 @@ public static class ConfKeys {
         public static final String MAX_ROWS_IN_MEMORY = "druid.memory.max_rows";
         public static final String SEGMENT_ROLLUP = "druid.segment.rollup";
         public static final String USE_DEFAULT_VALUES_FOR_NULL = "druid.use_default_values_for_null";
+        public static final String USE_THREE_VALUE_LOGIC_FOR_NATIVE_FILTERS = "druid.use_three_value_logic_for_native_filters";
```

Comment:
can you please add this to https://github.com/rovio/rovio-ingest?tab=readme-ov-file#optional-properties